refactor/ast.py

```diff
@@ -255,5 +255,57 @@ def retrieve_segment(self, node: ast.AST, segment: str) -> None:
                 self.fill()
             self.write(segment)
 
+class PreciseEmptyLinesUnparser(PreciseUnparser):
```

Comment:
I don't think people would actively change their unparsers (unless speed is a concern for them, which is why we have precise and fast). I'd believe this sort of improvement can go into the precise one directly (instead of a custom one).

Reply:
This will break the backward compatibility. For example the test: `test_precise_unparser_comments()` will now fail as the empty lines will no longer be removed, as well as the aligned comment separated with an empty line
![image](https://user-images.githubusercontent.com/93164135/209566137-4b0ec9da-e826-42c0-9dec-cda98aaf217c.png)

Proposing to add an argument to the parser to respect the backward compatibility